code-challenges.js

```diff
@@ -12,53 +12,86 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
-
-
-
+const divideByThree = arg =>{
+    if(arg%3 === 0){
+        console.log(`${arg} is divisible by three`);        
+    } else if (arg%3 !== 0){
+        console.log(`${arg} is not divisible by three`);
+        
+    }
+}
+
+divideByThree(num1);
+divideByThree(num2);
+divideByThree(num3);
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+const capitalize = arg => {
+    let newArray = arg.map(word =>
+        word.replace(/^\w/, c => c.toUpperCase()))   
+    return newArray;
+    }
 
-
+console.log(capitalize(randomNouns));
 
 
 // --------------------3) Create a function that takes in the mixedDataArray array and returns an array with ONLY NUMBERS sorted from least to greatest.
 
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
+const onlyNumbers = arg => {
+    let newArray = arg.filter(number => typeof number === "number").sort((a,b) => a - b);
+    return newArray;
+}
 
-
-
+console.log(onlyNumbers(mixedDataArray));
 
 // --------------------4) Create a function that takes in a string and logs the index of the first vowel.
 // Use the test variables provided.
 
 var vowelTester1 = "learn"
 // Expected output: 1
-var vowelTester1 = "throw"
+var vowelTester2 = "throw"
 // Expected output: 3
 
+const findVowel = arg =>{
+    let pos = arg.search(/[aeiou]/gi);
+    return pos;
```

Comment:
this is very concise and lovely! but, same as above, you can just return the first line ^^. also reiterating from above, be sure to comment what's actually happening in the code. interviewers may also ask you not to use "shortcuts" and write out the long form so be prepared for that just in case.